specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2022-11-01-preview/examples/DevToolPortals_CreateOrUpdate.json

```diff
@@ -51,15 +51,20 @@
           },
           "url": "aaa.com",
           "provisioningState": "Succeeded",
```

Comment:
The changes in this file seem like they reflect breaking changes to existing preview API versions, am I understanding that correctly?

Reply:
Never mind, I see you changed these in a later commit.